js/common/tests/http/document-babies-spec.js

```diff
@@ -1208,4 +1208,77 @@ describe('babies collection document', function () {
       expect(b7[0]._oldRev).to.be.a('string');
     });
   });
+
+  describe('overwrite', function () {
+    let base_url = '/_api/document/' + cn;
+    it('overwrite once', function () {
+      let url1 = base_url;
+      let req1 = request.post(url1, extend(endpoint, {
+        body: JSON.stringify([{
+          'Hallo': 12
+        }])
+      }));
+      let b1 = JSON.parse(req1.rawBody);
+      let res1 = b1[0];
+
+      let url2 = base_url + '?overwrite=true&returnOld=true';
+      let req2 = request.post(url2, extend(endpoint, {
+        body: JSON.stringify([{
+          '_key': res1._key,
+          'ulf': 42
+        }])
+      }));
+      let b2 = JSON.parse(req2.rawBody);
+      let res2 = b2[0];
+
+      expect(req2.statusCode).to.equal(202);
+      expect(req2._key).to.equal(req1._key);
```

Comment:
"old" is not checked here.
It is not checked if the document later has the correct values in it.